source/partials/_signup-cta.haml

```diff
@@ -0,0 +1,6 @@
+- cta_label ||= 'Sign Up'
+- cta_placeholder ||= 'Email'
+%a.btn.btn-primary{ href: 'https://dashboard.aptible.com/signup' }= cta_label
+-# %form.signup-cta.single-line{ action: 'http://localhost:4200/signup' }
+-#   %input.single-line__input{ type: 'email', placeholder: cta_placeholder }
+-#   %button.single-line__submit{ type: 'submit' }= cta_label
```

Comment:
Noting here that the single line email input is styled and ready... Just need to test the dashboard side.